core/src/main/scala/cats/Parallel.scala

```diff
@@ -0,0 +1,139 @@
+package cats
+
+import cats.arrow.FunctionK
+
+/**
+  * Some types that form a Monad, are also capable of forming an Applicative that supports parallel composition.
+  * The Parallel type class allows us to represent this relationship.
+  */
+trait Parallel[M[_], F[_]] extends Serializable {
+  /**
+    * The applicative instance for F[_]
+    */
+  def applicative: Applicative[F]
+
+  /**
+    * Natural Transformation from the parallel Applicative F[_] to the sequential Monad M[_].
+    */
+  def sequential(implicit M: Monad[M]): F ~> M
```

Comment:
Why do we need to accept this implicit? Why not make `def monad: Monad[M]` on Parallel?

Reply:
I initially only did this to make instances of `Parallel` shorter, but honestly, I really don't feel strongly about it either way.